labs/yui.md

```diff
@@ -0,0 +1,78 @@
+
+In addition, we aim to apply the method not only to the permissioned chains, but to the public chains as well.
+
+### Provide a unified communication method independent from a specific blockchain/DLT implementation.
+This enables application developers to develop chain agnostic protocols.
+
+### Support arbitrary data transfer and computation
+Developer can design and implement the specific protocol for its cross-chain application, not limited to token transfer use case.
+
+### Avoid adding components that introduce additional trust (apart from individual blockchains)
+Introducing such components may decrease the level of security of a system as it would be bounded by the lowest component in the system.
+
+Also, introducing new (off-chain) components with high fault tolerance will also lead to increased operational costs
+
+### Do not introduce a new layer
+Each actor (of ledgers) has only to interact with a respective ledger of its interest to complete a cross-chain operation unless the application has additional requirements.
+
+## Use cases
+Cross-chain smart contracts for arbitrary data and procedures, not limited to atomic swap of tokens.
+- Cross-chain token transfer
+- Cross-chain Atomic swap
+  - Example includes delivery versus payment settlement
+- Cross-chain atomic execution of arbitrary procedures
+
+## Current status and future plan
+We have already developed modules for Hyperledger Fabric, Hyperledger Besu, and Corda. We plan to apply these modules for practical applications as well as to expand support for other blockchains.
+- At first, we are focusing on permissioned blockchain environment
+
+Aim to enable developers in Hyperledger community to introduce interoperability in their systems and develop cross-chain applications without too many changes and increase the complexity
+
+## Components
+Client (on-chain) module and Relayer to support various ledgers in align with the design principles
+- Client module should be designed to support each underlying ledger
+
+Various software to support the development of cross-chain systems
+- Modules to support the development of cross-chain contracts
+  - Implementation of an atomic commit protocol for cross-chain atomic operation
+  - Implementation of authentication for cross-chain contracts
+- System to support the coordination of cross-chain contract and applications
+  - Explorer to track the cross-chain status
+
+
+# Initial Committers
+- https://github.com/bluele
+- https://github.com/3100
+- https://github.com/siburu
+
+
+# Pre-existing repository
+- https://github.com/datachainlab/fabric-ibc
+- https://github.com/datachainlab/ibc-solidity
+- https://github.com/datachainlab/corda-ibc
+
+
+# Contributors
+- https://github.com/storiumi
+- https://github.com/haifengxi
+- https://github.com/harrietcao
+- https://github.com/zhangyelong
+- https://github.com/HuLaTown
```

Comment:
I do not understand what this section is? Are these people who should be added to the lab to be able to commit? If so, this is what the `Initial Committers` section is.

Reply:
Thanks Tracy,
Two questions;
- Should initial committers represent people who can commit repos afterwards? In that case, can we add committers by ourselves later on?
- I assume there could be community work such as replying questions, facilitating meetings and so forth apart from the maintaining the codes. Is there any way to list contacts for this purpose?
Thanks.

Reply:
Yes, you can add more contributors afterwards and that's in your control.
Your repo should have files such as README.md and MAINTAINERS.md in which you are more than welcome to add categories and roles as you see fit.

Reply:
Thanks Arnaud,
I will remove contributors from the file to just list initial committers.
And, I will make maintainers.md to list contributors afterwards in the repository.